src/app/mosh_client.html

```diff
@@ -73,6 +73,10 @@
             <td>Mosh server command:</td>
             <td><input id="server-command" type="text" disabled placeholder="default"></td>
           </tr>
+          <tr id="mosh-escape-key-row">
+            <td>Mosh escape key (default: <code>^</code>)</td>
+            <td><input id="mosh-escape-key" type="text" disabled placeholder="default"></td>
```

Comment:
There's a little problem with this: MOSH_ESCAPE_KEY must be one of the ASCII characters from 1-127. It isn't possible to type many of those into a web input field. E.g., it would be impossible here to specify the default of Ctrl-^ if it weren't the default; imagine if you wanted to change that to Ctrl-%.

I don't have a great solution here. A simple one would be a giant drop-down with all 127 options, but that's messy. Another would be to permit users to specify control characters somehow, but then you'd have to write a parser. Another would be for the user to specify the ASCII code, but that's unintuitive for printable characters such as ~. Another would be a hybrid, whereby printable characters are entered literally, while the ASCII code is given for others; but that cannot disambiguate if, e.g., you wanted your escape key to be the number 1 (which is crazy, but who am I to judge?). So then there'd have to be a disambiguator. And then there's the issue of documenting how to do this if there's a special format. That messy drop-down is starting to sound a lot more appealing.

If you wanted to generate the drop-down in code since it will have so many values which can all be mechanically derived, that'd be fine.